src/uu/cp/src/cp.rs

```diff
@@ -353,6 +354,9 @@ fn show_debug(copy_debug: &CopyDebug) {
 const ABOUT: &str = help_about!("cp.md");
 const USAGE: &str = help_usage!("cp.md");
 const AFTER_HELP: &str = help_section!("after help", "cp.md");
+const SIZE_LIMIT_FOR_NORMAL_COPY: u64 = 524288000;
+const BUFFER_SIZE: usize = 4096 * 4;
```

Comment:
please document why * 4?

Reply:
I do not have very good evidence/reasoning here but based on standard file system block size, this ought to be good number.